test/WebJobs.Script.Tests.Integration/Management/InstanceManagerTests.cs

```diff
@@ -79,6 +79,7 @@ public async Task StartAssignment_AppliesAssignmentContext()
             Assert.Collection(logs,
                 p => Assert.StartsWith("Starting Assignment", p),
                 p => Assert.StartsWith("Applying 1 app setting(s)", p),
+                p => Assert.StartsWith($"Will be using  app setting as zip url", p),
```

Comment:
Looks like we have 2 spaces between "using app"

Reply:
This is the case where no run from package url is specified. The environment variables is returned as an empty string.